data/distro.yml

```diff
@@ -48,6 +48,12 @@
           <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
         </code></pre>
       </p>
+      <p>Fedora has a "filtered" Flathub remote added but disabled. It needs to be removed and added again to get the full repository when you enable it.
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-delete flathub 
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Comment:
I may be wrong, but I think that removing the `--if-not-exists` switch could also help - instead of using flatpak remote-delete.